homeassistant/components/rainmachine/sensor.py

```diff
@@ -61,6 +68,7 @@ class RainMachineSensorEntityDescription(
         icon="mdi:water-pump",
         native_unit_of_measurement="index",
         entity_registry_enabled_default=False,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
What kind of sensor is this? Is it a measurement?

Reply:
Good call: this isn't a value that we care about measuring over time.